packages/components/src/components/Autocomplete/index.ts

```diff
@@ -1 +1,11 @@
-export * from "./Autocomplete";
+import { Autocomplete as _Autocomplete } from "./Autocomplete";
+import { MultiSelectAutocomplete } from "./MultiSelectAutocomplete";
+
+type AutocompleteFamily = typeof _Autocomplete & {
+    MultiSelect: typeof MultiSelectAutocomplete;
+};
+
+const Autocomplete = _Autocomplete as AutocompleteFamily;
+Autocomplete.MultiSelect = MultiSelectAutocomplete;
```

Comment:
I think we should rename this to be `Autocomplete.Multiselect` (no capitalization of "s") since it seems to the be written in one word